x-pack/plugins/enterprise_search/public/applications/app_search/components/log_retention/log_retention_logic.test.ts

```diff
@@ -267,23 +252,42 @@ describe('LogRetentionLogic', () => {
     });
 
     describe('fetchLogRetention', () => {
+      beforeAll(() => jest.useFakeTimers());
+      afterAll(() => jest.useRealTimers());
+
+      describe('isLogRetentionUpdating', () => {
```

Comment:
[optional] This is totally trivial, but you probably don't need to wrap this in a describe, since there's only one test and it seems to be descriptive enough on it's own.

Reply:
👍 I'm good with removing it! I think this test was in here from before in https://github.com/elastic/kibana/commit/c1978e5584424af383365269eb394444703cf33e so I basically just mindlessly reverted it, lol. Happy to reduce lines

Reply:
Actually I just looked through the file and it looks like this is an older/more verbose logic test where multiple listeners have a `describe('isLogRetentionUpdating')` block:

https://github.com/elastic/kibana/blob/a111b6a2870bc17760d99aa470613bb3106fe563/x-pack/plugins/enterprise_search/public/applications/app_search/components/log_retention/log_retention_logic.test.ts#L110-L111

https://github.com/elastic/kibana/blob/a111b6a2870bc17760d99aa470613bb3106fe563/x-pack/plugins/enterprise_search/public/applications/app_search/components/log_retention/log_retention_logic.test.ts#L220-L221

If no objection I'm going to leave this as-is for consistency w/ the rest of the file, if we want to come back in here later and make our describe/it blocks a bit more succinct to match our newer logic tests I'm def down for that

Reply:
NP!